database.json

```diff
@@ -982,6 +982,12 @@
     "repo": "dodecasaurus",
     "desc": "Dice utility that makes generating random dice values simple and convenient."
   },
+  "domelementtype": {
+    "type": "github",
+    "owner": "DenoBRComunitty",
+    "repo": "domelementtype",
+    "desc": "All the types of dom for Deno"
+  },
```

Comment:
put this after dom_serializer.

Reply:
oh wait you already did

Reply:
resolved, I changed the name of the module I added in https://github.com/denoland/deno_website2/pull/887